pkg/controller/master/image/vm_image_controller.go

```diff
@@ -127,6 +129,16 @@ func (h *vmImageHandler) createBackingImage(image *harvesterv1.VirtualMachineIma
 		bi.Spec.SourceParameters[types.DataSourceTypeDownloadParameterURL] = image.Spec.URL
 	}
 
+	if image.Spec.SourceType == harvesterv1.VirtualMachineImageSourceTypeExportVolume {
+		pvc, err := h.pvcCache.Get(image.Namespace, image.Spec.VolumeName)
```

Comment:
We cannot use the image namespace to get PVC. The PVC namespace may be different from the image namespace.

Reply:
@gitlawr updated